pkg/ccl/backupccl/backupdestination/backup_destination_test.go

```diff
@@ -6,7 +6,7 @@
 //
 //     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
 
-package backupccl
+package backupdestination
```

Comment:
Out of curiosity, do you know how far we are from being able to put this in `backupdestination_test`?

Reply:
not too far, done 😃